test/spec/modules/oneVideoBidAdapter_spec.js

```diff
@@ -593,13 +593,13 @@ describe('OneVideoBidAdapter', function () {
 
       it('should get correct user sync when iframeEnabled', function () {
         let pixel = spec.getUserSyncs({pixelEnabled: true}, {}, {gdprApplies: true, consentString: GDPR_CONSENT_STRING})
-        expect(pixel[2].type).to.equal('image');
-        expect(pixel[2].url).to.equal('https://sync-tm.everesttech.net/upi/pid/m7y5t93k?gdpr=1&gdpr_consent=' + GDPR_CONSENT_STRING + '&redir=https%3A%2F%2Fpixel.advertising.com%2Fups%2F55986%2Fsync%3Fuid%3D%24%7BUSER_ID%7D%26_origin%3D0&gdpr=1&gdpr_consent=' + encodeURI(GDPR_CONSENT_STRING));
+        expect(pixel[1].type).to.equal('image');
+        expect(pixel[1].url).to.equal('https://sync-tm.everesttech.net/upi/pid/m7y5t93k?gdpr=1&gdpr_consent=' + GDPR_CONSENT_STRING + '&redir=https%3A%2F%2Fpixel.advertising.com%2Fups%2F55986%2Fsync%3Fuid%3D%24%7BUSER_ID%7D%26_origin%3D0&gdpr=1&gdpr_consent=' + encodeURI(GDPR_CONSENT_STRING));
```

Comment:
Not Important , just a suggestion:
It would be good if we check for pixel[0].url / pixel[2].url  to as you are actually changing those pixels.
like below:
`expect(pixel[0].url).to.equal('https://pixel.advertising.com/ups/57304/sync?gdpr=&gdpr_consent=&_origin=0&redir=true')`

Reply:
Understood and thank you for the review @prasad457!
Updated tests accordingly.